derive_builder/src/options/struct_mode.rs

```diff
@@ -1,15 +1,18 @@
 use syn;
-use options::{OptionsBuilder, OptionsBuilderMode, parse_lit_as_string, FieldMode, StructOptions};
+use options::{OptionsBuilder, OptionsBuilderMode, parse_lit_as_string, parse_lit_as_bool, parse_lit_as_path, FieldMode, StructOptions};
 use derive_builder_core::{DeprecationNotes, Bindings};
 
 #[derive(Debug, Clone)]
 pub struct StructMode {
+    build_fn_name: Option<String>,
+    build_fn_enabled: bool,
```

Comment:
I don't know very much about this code base yet, so this is more a question than feedback: Could this not be modeled in only one field? If the `Option<String>` is `None` that means we don't generate a build method. So whoever creates the defaults for this default to `Some("build")` instead of `None`?